tests/Controller/ExportControllerTest.php

```diff
@@ -307,6 +307,25 @@ public function testXmlExport()
         $this->assertNotEmpty('updated_at', (string) $content->entry[0]->updated_at);
     }
 
+    public function testMdExport()
+    {
+        $this->logInAs('admin');
+        $client = $this->getTestClient();
+
+        ob_start();
+        $crawler = $client->request('GET', '/export/all.md');
+        ob_end_clean();
```

Comment:
I don't get why ob_* functions and a $crawler variable are needed here, I think they can be removed, just keep the call to request